openpype/hosts/maya/plugins/publish/extract_camera_mayaScene.py

```diff
@@ -219,3 +238,40 @@ def process(self, instance):
 
         self.log.debug("Extracted instance '{0}' to: {1}".format(
             instance.name, path))
+
+    @classmethod
+    def get_attribute_defs(cls):
+        defs = super(ExtractCameraMayaScene, cls).get_attribute_defs()
+
+        defs.extend([
+            BoolDef("keep_input_connections",
+                    label="Keep Input Connections",
+                    tooltip="Preserving input connections will allow Image Planes to stay connected on publish",  # noqa
+                    default=cls.keep_input_connections),
+
+        ])
+
+        return defs
+
+    def ensure_image_plane_attachment(self, camera, reattach=False):
+        """Reattaches image planes to baked or original cameras.
+
+        Baked cameras are duplicates of original ones, even with
+        `keep_input_connections` the connection of image plane is still on the
+        original one. This attaches it to duplicated camera properly.
+        If 'reattach' it attaches back image plane to original camera.
+        TODO figure out use of some context manager
+        (`delete_after` doesnt work)
+        """
+        image_planes = cmds.listConnections(camera, type="imagePlane")
+        if not image_planes:
+            return
+        if reattach:
+            # find original camera name
+            camera = camera[0:camera.find("_baked")]
+
+        image_planes = [x.split("->")[1] for x in image_planes]
```

Comment:
Be aware that it's not required for an image plane path to include `->` the image plane can be connected to the camera and NOT be parented to it as far as I know. As such, I believe what you need to do is: `x.rsplit("->", 1)[-1]` so that even if `->` is not there you get a valid result.

Reply:
Updated